src/tree.h

```diff
@@ -172,6 +172,7 @@ struct policy_node {
 	union node_data data;
 	char *exceptions;
 	unsigned int lineno;
+	unsigned char flagged;
```

Comment:
I like making it generic if it's going in the node structure, but I'm worried about whether it will actually be useful beyond this one case, since there's no way of knowing what flagged it.  I'm wondering if a string_list where you added the check id of what it's being flagged for would have more potential for being useful in the future?

The way this sort of thing is handled for other checks is by adding a struct to node_data for that particular node flavor, and then adding a granular flag like "bare_module" there.

Reply:
Added example commit

Reply:
Yeah, I like this new version better.